src/app.jsx

```diff
@@ -320,9 +331,16 @@ class Application extends React.Component {
         /* The following events do not need to trigger any state updates */
         case 'attach':
         case 'exec':
+        case 'exec_died':
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1353-20230713-111150-a8183c4d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)